src/main/java/org/apache/mesos/offer/TaskUtils.java

```diff
@@ -1,223 +1,282 @@
 package org.apache.mesos.offer;
 
-import java.util.*;
-
 import com.google.protobuf.ByteString;
 import org.apache.mesos.ExecutorDriver;
 import org.apache.mesos.Protos;
-import org.apache.mesos.Protos.Label;
-import org.apache.mesos.Protos.Labels;
-import org.apache.mesos.Protos.TaskID;
-import org.apache.mesos.Protos.TaskInfo;
-import org.apache.mesos.Protos.TaskStatus;
+import org.apache.mesos.Protos.*;
 import org.apache.mesos.config.ConfigStore;
+import org.apache.mesos.specification.ResourceSpecification;
+import org.apache.mesos.specification.TaskSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
 
 /**
  * Various utility methods for manipulating data in {@link TaskInfo}s.
  */
 public class TaskUtils {
-  private static final String TARGET_CONFIGURATION_KEY = "target_configuration";
```

Comment:
concat `?w=0` to see only non-whitespace changes.  I'm moving everything to 4 spaces as I encounter it.